.changeset/link-add-underline.md

```diff
@@ -0,0 +1,10 @@
+---
+"@salt-ds/core": minor
+---
+
+Added `underline` prop to `Link`. It controls when does underline text decoration applied to link.
```

Comment:
Added `underline` prop to `Link`. It controls when underline text decoration is applied to a link.